manual/integrate_root_into_my_cmake_project/index.md

```diff
@@ -55,36 +60,33 @@ Generate dictionary for a set of header files (convenient wrapper on top of the
       [OPTIONS opt1 opt2 ...] )
 {% endhighlight %}
 
-The `${dictionary}.cxx` and `${dictionary}.pcm` files are created using the headers and the selection file, calling the `genreflex` command. Optionally, you can provide `OPTIONS` to customize the way the `genreflex` command is called.
-
 ## Full example (event project)
 
 The following is an example of a project that creates a library including a dictionary and an executable file.
 
 {% highlight C++ %}
-# CMakeLists.txt for event package. It creates a library with a dictionary and a main program.
+# CMakeLists.txt for the "event" package. It creates a library with a dictionary and a main program.
 cmake_minimum_required(VERSION 3.0 FATAL_ERROR)
 project(event)
 
-# You need to tell CMake where to find the ROOT installation. This can be done in a number of ways:
-#   - ROOT built with classic configure/make use the provided $ROOTSYS/etc/cmake/FindROOT.cmake.
-#   - ROOT built with CMake. Add in CMAKE_PREFIX_PATH the installation prefix for ROOT.
-list(APPEND CMAKE_PREFIX_PATH $ENV{ROOTSYS})
+# If ROOT is not installed in a default system location you need to tell CMake where to find it.
+# Sourcing `thisroot.sh` already sets the required environment variables.
+# Otherwise, adding the installation prefix of ROOT to CMAKE_PREFIX_PATH tells cmake where to look:
+#list(APPEND CMAKE_PREFIX_PATH "/path/to/root/installation")
```

Comment:
Isn't a better suggestion to add `-DROOT_DIR="/path/to/root/installation"` to the command line invocation, rather than suggesting it be hardcoded it into a CMakeLists?

Reply:
I'd say even better would be to suggest `-DCMAKE_PREFIX_PATH=...`  rather than `-DROOT_DIR=...`, as that's a generic way that works for all packages.

Reply:
how about now?

Reply:
Now you overwrite `CMAKE_PREFIX_PATH` with a cached variable so the CMakeLists.txt can't set it. While it's not the best practice, some CMakeLists do mess with it - it would be better to be able to set the directory per package in the examples for those packages, and `<package_name>_DIR` is generic and works for all packages.

Reply:
if I understand correctly the discussion is now about `-DCMAKE_PREFIX_PATH` vs `-DROOT_DIR`. I don't really know enough to have a preference, @amadio @oshadura what do you say?

Reply:
I think Henri's argument is good. Go for `-DROOT_DIR=...` then.

Reply:
alright, `-DROOT_DIR` it is then! done!